### ISSUE TITLE
Stray `LICENSE` installed into site-packages

### ISSUE THREAD

**Issue by mgorny** [cunla/fakeredis-py#320]
**Describe the bug**
The package installs a stray `LICENSE` file to the top directory of site-packages.

**To Reproduce**
```
$ pip install fakeredis
Collecting fakeredis
  Downloading fakeredis-2.24.0-py3-none-any.whl.metadata (3.8 kB)
Collecting redis>=4 (from fakeredis)
  Using cached redis-5.0.8-py3-none-any.whl.metadata (9.2 kB)
Collecting sortedcontainers<3,>=2 (from fakeredis)
  Using cached sortedcontainers-2.4.0-py2.py3-none-any.whl.metadata (10 kB)
Downloading fakeredis-2.24.0-py3-none-any.whl (97 kB)
Using cached redis-5.0.8-py3-none-any.whl (255 kB)
Using cached sortedcontainers-2.4.0-py2.py3-none-any.whl (29 kB)
Installing collected packages: sortedcontainers, redis, fakeredis
Successfully installed fakeredis-2.24.0 redis-5.0.8 sortedcontainers-2.4.0
$ ls $VIRTUAL_ENV/lib/python*/site-packages
fakeredis                   LICENSE  pip-24.2.dist-info  redis-5.0.8.dist-info  sortedcontainers-2.4.0.dist-info
fakeredis-2.24.0.dist-info  pip      redis               sortedcontainers
```

Note the `LICENSE` file that shouldn't be there.

**Expected behavior**
All files installed into `fakeredis` directory.

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - OS: Gentoo Linux amd64
 - python version: 3.12.5
 - redis-py version: 2.24.0
 - full requirements.txt? n/a

**Additional context**
Apparently this is because `LICENSE` is listed in `tool.poetry.packages`.

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/cunla) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/cunla/fakeredis-py/issues/320">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/cunla/fakeredis-py/issues/320/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/cunla/fakeredis-py/issues/320/pledge.svg">
</picture>
</a>

**Comment by cunla** [comment]
thanks for reporting this. Fixed in 2.24.1

**Comment by mgorny** [comment]
Thanks.
